### ISSUE TITLE
Add "Accessibility Mode" to OLH and Material

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4946]
1.9 has prioritised making Clean accessible, due to limitations in the frameworks used by Material and OLH.

We already have a development mode that enables easy switching between themes. The proposal from recent design meeting was to make something similar avaliable to users (both logged in and not) when on journals that use Material or OLH. Then if if a user encounters and accessibility barrier, they could choose to use a more accessible version.

This should work for users regardless of whether they are logged in.

**Comment by StephDriver** [comment]
- Accessibility Mode on/off switch on every page, and near the top.
  - either as the next option after skip to main
  - or top right part of the header, as with hourglass (black button) 
  - will need an adjacent link to the A11y page (which can be updated to a link directly to explaining A11y mode as part of #5289).
- As previously discussed, a11y mode should remain on for the session.
- Are the three Clarity colour themes all user-selectable? If so, consider how selection is handled as part of the on switch.
- Mindful of how press and journal sites don't always have the same theme,  the implementation should be as similar as possible across the themes, so that accessibility mode switch is very easy to find, regardless of difficulties when not in that mode.

<img width="387" height="77" alt="Image" src="https://github.com/user-attachments/assets/f1ef72b3-357c-4634-a23b-f2f54bd7bdff" />

Implementation similar to hourglass a11y animation switch could work nicely:

<img width="379" height="109" alt="Image" src="https://github.com/user-attachments/assets/2050a619-93f5-45fe-a64f-59cb212ff2ac" />

If using the button as per hourglass, there should be a change that is not colour alone, to indicate whether a11y mode is on/off, in the same location.  So I suggest that in Clarity, where this would then be an off button, it looks distinct to how it is in the other themes.  

Note:

The implementation in hourglass for an Accessibility menu item on mobiles, would need modifying to work on Janeway because we have an accessibility link in the footer - and this could give us two links of same name with different actions.

<img width="494" height="314" alt="Image" src="https://github.com/user-attachments/assets/abaacdc1-1ccc-4d62-af4b-85e093d77c1e" />

**Comment by mauromsl** [comment]
Closed by https://github.com/openlibhums/janeway/pull/5027

hourglass work moved to https://github.com/openlibhums/hourglass/issues/471
